src/components/pageNav/index.jsx

```diff
@@ -42,13 +43,14 @@ class PageNav extends React.PureComponent {
   }
 
   render() {
-    const { previous, next, baseUrl } = this.props;
+    const { previous, next, baseUrl, disableButton } = this.props;
 
     return (
       <div className={styles.pageNavigation}>
         <nav role="navigation" className={styles.navSecondary}>
-          {previous && this.renderNavItem('previous', previous, baseUrl)}
-          {next && this.renderNavItem('next', next, baseUrl)}
+          {previous &&
+            this.renderNavItem('previous', previous, baseUrl, disableButton)}
```

Comment:
see related changes; this becomes: `{previous && this.renderNavItem('previous', previous, baseUrl)}`